src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,106 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node<K,V>[] hashMapTable;
+    private int size;
```

Comment:
You may add threshold and current capacity. It will simplify

Reply:
I'd like to show this way to mentors